benchkit/hdc/__init__.py

```diff
@@ -83,7 +83,7 @@ def _devices() -> Iterable[str]:
         Returns:
             Iterable[HDCDevice]: list of devices recognized by hdc.
         """
-        output = OpenHarmonyDeviceConnector._host_shell_out(command="hdc list targets")
+        output = OpenHarmonyDeviceConnector._host_shell_out(command="hdc.exe list targets")
```

Comment:
The `.exe` suffix should be conditiional if benchkit is being run on a windows host.
Since hdc is required in multiple places perhaps make a function like hdc_path which also searches for hdc in PATH an returns a helpful error if hdc is not in PATH. (I think python should have a builtin function for that)

Reply:
that would be a binary dependency for this class?

Reply:
`hdc` is basically our `adb` equivalent, so yes `hdc` is required, same as `adb` is required for talking with android devices.

Reply:
you can list dependencies in a method like in benchmark subclasses, then we expect these dependencies to be eventually checked before running the benchmark